addons/rating/models/rating_mixin.py

```diff
@@ -61,6 +61,7 @@ class RatingMixin(models.AbstractModel):
     rating_count = fields.Integer('Rating count', compute="_compute_rating_stats", compute_sudo=True)
     rating_avg = fields.Float("Rating Average", compute='_compute_rating_stats', compute_sudo=True)
     rating_percentage_satisfaction = fields.Float("Rating Satisfaction", compute='_compute_rating_satisfaction', compute_sudo=True)
+    rating_text = fields.Selection(string="Rating Text", groups='base.group_user', related="rating_ids.rating_text")
```

Comment:
I think we select the last rating to get the `rating_text` field, like the others related fields in this mixin. So we could rename it by `rating_last_text`.
```suggestion
    rating_last_text = fields.Selection(string="Rating Text", groups='base.group_user', related="rating_ids.rating_text")
```